packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -0,0 +1,184 @@
+    /**
+     * The content for the card.
+     */
+    children: React.ReactNode;
+    /**
+     * A set of localizable labels for this component, including a dismiss button
+     * and the card itself, if marked as an HTML region.
+     */
+    labels?: {
+        dismissButtonAriaLabel?: string;
+        cardAriaLabel?: string;
+    };
+    /**
+     * A callback function to handle dismissing the card. When this prop is present,
+     * a dismiss button with an X icon will be rendered.
+     */
+    onDismiss?: (e?: React.SyntheticEvent) => void;
+    /**
+     * An optional attribute to remove this component from the accessibility tree
+     * and keyboard tab order, such as for inactive cards in a stack.
+     */
+    inert?: boolean;
+    /**
+     * The test ID used to locate this component in automated tests.
+     */
+    testId?: string;
+} & AllowedStyleProps;
+
+/**
+ * The Card component is a flexible, reusable UI building block designed to
+ * encapsulate content within a structured, visually distinct container.
+ * Its primary goal is to present grouped or related information in a way that
+ * is visually consistent, easily scannable, and modular across different
+ * parts of the application.
+ *
+ * Cards provide a defined surface area with clear visual boundaries
+ * (via border-radius and box-shadow elevation tokens), making them ideal for
+ * use cases that involve displaying comparable content items side-by-side or
+ * in structured layouts such as grids, lists, or dashboards.
+ *
+ * Note: cards do not set a default width. Width styles should be set by the consumer
+ * with the `styles.root` prop.
+ *
+ * ### Usage
+ *
+ * ```jsx
+ * import {Card} from "@khanacademy/wonder-blocks-card";
+ *
+ * <Card>
+ *   <Heading>This is a basic card.</Heading>
+ * </Card>
+ * ```
+ */
+
+const Card = React.forwardRef(function Card(
+    props: Props,
+    ref: React.ForwardedRef<any>,
+) {
+    const {
+        styles,
+        labels,
+        tag,
+        testId,
+        backgroundColor = "default",
+        borderRadius = "radius_080",
+        padding = "size_160",
+        children,
+        onDismiss,
+        inert,
+    } = props;
+
+    const componentStyles = getComponentStyles({
+        backgroundColor,
+        borderRadius,
+        padding,
+    });
+    return (
+        <View
+            aria-label={labels?.cardAriaLabel}
+            style={[componentStyles.root, styles?.root]}
+            ref={ref}
+            tag={tag}
+            testId={testId}
+            {...{inert: inert ? "" : undefined}}
+        >
+            {onDismiss ? (
+                <DismissButton
+                    aria-label={labels?.dismissButtonAriaLabel || "Close"}
+                    onClick={(e) => onDismiss?.(e)}
+                />
+            ) : null}
+            {children}
+        </View>
+    );
+});
+
+const getComponentStyles = ({
+    backgroundColor,
+    borderRadius,
+    padding,
+}: AllowedStyleProps) => {
```

Comment:
Something I started to do in WB was to create explicit styles for variants instead of creating dynamic styles since _I think_ we won't be able to do things as dynamically with css modules. My hope too is that explicitly laying out the styles will make it easier to code mod to css modules when the time comes! 

(These are my assumptions though, curious is either you or @jandrade have thoughts on this too!)

Reply:
I think I'll opt to do a refactor later on so this work can keep moving. Good call, though!